baselines/showcase/src/v1beta1/messaging_client.ts.baseline

```diff
@@ -1874,10 +1856,10 @@ export class MessagingClient {
    *   The promise has a method named "cancel" which cancels the ongoing API
    * call.
    *
-   * @example
+   * ```@example
```

Comment:
The backticks should be only around the code sample, not the tag.

Reply:
will do!

Reply:
The closing back ticks should be on a new line, at least that's what all the examples have. See here: https://tsdoc.org/pages/tags/example/